src/utils/dockerDirManager.ts

```diff
@@ -272,6 +272,9 @@ export default class DockerDirManager {
     if (currentCredsStore && await this.credHelperWorking(currentCredsStore)) {
       return currentCredsStore;
     }
+    if (process.env.CIRRUS_CI && this.credHelperWorking('none')) {
```

Comment:
Shouldn't we be using `CI` instead of `CIRRUS_CI` here? (That is: if we're running on GitHub Actions / GitLab / whatever, we still want to use the `none` credential helper.)

Reply:
Sure.

Reply:
Oh no, this is wrong. We want the unit tests run on CI to use the platform-specific tests. FWICT `CIRRUS_CI` is our only indication of code that is run on a CI system while doing E2E tests.